app/test/tokumei/close_connection_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Tokumei.CloseConnectionTest do
+  use ExUnit.Case
+
+  alias Raxx.Response
+  alias Raxx.Request
+
+  use Tokumei.CloseConnection
+
+  def handle_request(%{method: :GET}, _), do: Response.ok("GET")
+
+  test "sends back connection: close if that was received" do
```

Comment:
No don't think in this repo already. something like what is already set up in here is probably good https://github.com/CrowdHailer/Ace/blob/master/extensions/http(s)/test/ace/http/request_test.exs